src/gui/gui/run_gui.py

```diff
@@ -0,0 +1,39 @@
+import subprocess
+import pathlib
+import os
+
+####################
+# This file was created taking into account that it will be run with
+#   the following command: ros2 run gui run_gui
+# If it is run any other way it probably won't work
+# (Needs to be built with colcon and sourced beforehand)
+####################
+
+# Set folder names for both repos
+VICE_REPO_NAME="ssl-VICE"
+GUI_REPO_NAME="ssl-gui"
+
+# Get the path to the file being executed
+path_to_repo_parent_folder = pathlib.Path(__file__).parent.absolute()
+
+# Find the parent folder to the VICE repo
+while not (path_to_repo_parent_folder / VICE_REPO_NAME).exists():
```

Comment:
This loop to find the parent folder might not be necessary, because it seems to me that this loops aims to find the repo in any parent folder. But  it might cause problems if the expected folder isn't in the right place and this doesn't return a error.